scss/templates/user.scss

```diff
@@ -198,3 +181,105 @@
     }
   }
 }
+
+// User Navigation
+.user-secondary-navigation {
+  .new-private-message {
+    @include user-profile-btn;
+    width: 9rem;
+    margin-bottom: 1rem;
+  }
+
+  .nav-stacked {
+    li {
+      margin-bottom: 1.4375rem;
+
+      a {
+        color: $gray-100;
+        font-family: Libre Franklin;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 18px;
+        line-height: 22px;
+        padding-top: 0;
+        padding-bottom: 0;
+      }
+
+      a.active {
+        color: $blue;
+      }
+    }
+  }
+}
+
+.user-content-wrapper .user-additional-controls button {
+  @include user-profile-btn;
+  width: 9rem;
+
+  @include media-breakpoint-up(sm) {
+    margin-left: 0.8125rem;
+  }
+}
+
+// Notifications section
+.notification-history .notification {
+  box-shadow: $card-box-shadow;
+  padding: 0;
+
+  &:not(:last-child) {
+    margin-bottom: 0.75rem;
+  }
+
+  &,
+  &.unread {
+    background: $beige;
+  }
+
+  li a {
+    position: relative;
+
+    .d-icon {
+      color: $gray;
+      margin: 1.5rem;
+      width: 1.125rem;
+      height: 1rem;
+    }
+
+    div {
+      height: 100%;
+      padding: 0 1.5rem;
+      color: $blue;
+
+      span {
+        &:first-child {
+          color: $gray;
+        }
+
+        &:last-child {
+          color: $blue;
+          margin-left: 0.25rem;
+        }
+      }
+
+      // This is creates a border between the icon and the text
+      &::before {
+        content: "";
+        height: 4rem;
```

Comment:
I see a lot of hardcoded dimensions that capture my attention. 
Ideally you want to strive for dimensions that are "fluid" to the content. for instance, why here is not `height: 100%`? or why here the element itself with `display: block;` doesn't take the whole space? 

This will allow to change the height of the card without brake the component styles.

Reply:
Good catch, this was for testing porpoises, I should have removed that